governance.md

```diff
@@ -95,9 +137,14 @@ The main goal is not to get full agreement from all SC members on a final decisi
 
 Voting on a decision proposal should be used as a last resort solution, as it can potentially leave several SC members major concerns unaddressed.
 
+Some procedural decisions, such as expansion, can be approved without an SC meeting. This is done via GitHub and requires approval by
+2/3rds of SC members. If an SC member feels that further discussion is required a meeting can be called, even if the PR has otherwise been approved.
+If a meeting is called, the above procedure should be followed.
```

Comment:
not critical to add but maybe something to think about. in CNCF circles I've seen two types of votes: binding and non-binding where the latter is used by the community members to show signs of support. maybe this could be mentioned in the commit message at least that non-binding votes are also encouraged?